### ISSUE TITLE
Forgotten pulls #1675 #1651

### ISSUE THREAD

**Issue by qiancy98** [microsoft/winget-pkgs#1941]
It seems that these two pull requests pass all checks but be forgot by reviewers.
#1651 
#1675

**Comment by denelon** [comment]
It looks like #1651 is blocked, and we just haven't gotten to #1675 yet.

**Comment by qiancy98** [comment]
> It looks like #1651 is blocked, and we just haven't gotten to #1675 yet.

Thank you. I ask this because, I traced for a while the merging time of the merged issues, and find almost all "Azure-Pipeline-Passed"-label packages are merged, even they are arised several days later than #1675. I assumed that you ckeck the pr's by time so I raised this problem.

Besides, when searching with the filter "is:pr is:open label:Azure-Pipeline-Passed created:<2020-06-15 ", I find that there are 11 pull requests that pass the pipeline, not blocked by issues, but review required. Some of them only have 3 comments (meaning no reviewer commented it except the bot)!

Thank you for helping my curious of wondering if they are forgetten.
